src/homework/day8/Doubles.java

```diff
@@ -21,17 +21,17 @@ public static void main(String[] args) {
         }
 
         // найти сумму всех дробных частей и вывести результат в консоль
-//        double sum = 0.00;
+//        double sum = 0.0000;
 //        double i = doubles - (int)doubles;
-//        for (double i = 0; i < myDoub.length; i++) {
-//            sum = sum + myDoub[i];
+//        for (double i = 0; i < myDoubles.length; i++) {
+//            sum = sum + myDoubles[i];
 //        }
 //        System.out.println("сумма всех дробных частей: " + sum);
 
         // Проитерировать список по индексу и вывести целые части в консоль через пробел
 //        int y = (int)doubles;
-//        for (int y = 0; y < myDoub.length; y++) {
-//            System.out.print((int)myDoub + " ");
+//        for (int y = 0; y < myDoubles.length; y++) {
+//            System.out.print((int)myDoubles + " ");
 //        }
```

Comment:
If you comment out lines of code. Please, add notes, why you do this or delete it. Because it's bad practice to leave your code in such kind of way.